CODEOWNERS

```diff
@@ -1 +1,9 @@
+stable/rbac-manager @sudermanjr @robscott
+stable/aws-iam-authenticator @sudermanjr @davekonopka
+incubator/autospotting @sudermanjr @coreypobrien
+incubator/fluentd @dosullivan @coreypobrien
+incubator/helm-release-pruner @davekonopka
+incubator/imagepullsecret-deploy @crmejia
+incubator/kubebench @mjhuber
+incubator/ro-cert-manager @sudermanjr @endzyme
```

Comment:
The [Github docs](https://help.github.com/en/articles/about-code-owners) seem to say that broad default should go first followed by specific overrides:

```# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.```